TicTacToe/src/tictactoe/TicTacToeAI.java

```diff
@@ -0,0 +1,275 @@
+/*
+ * To change this license header, choose License Headers in Project Properties.
+ * To change this template file, choose Tools | Templates
+ * and open the template in the editor.
+ */
+package tictactoe;
+/**
+ * Class contains the player's impossible AI opponent.
+ * 
+ * @author Evan
+ */
+public class TicTacToeAI
+{
+    int moveNum = 1;
+    public TicTacToeAI(){}
+    
+    public void AIMove(TicTacToe board)
```

Comment:
The AIMove method is enormous.  This should really have been broken up into smaller methods.